pkg/apis/istio/v1alpha2/istiocontrolplane_types.proto

```diff
@@ -382,7 +384,7 @@ message GalleyComponentSpec {
 message CommonComponentSpec {
     // Selects whether this component is installed.
     google.protobuf.BoolValue enabled = 1;
-    // Namespace that all resources for this component are installed into. Override for parent feature namespace.
+    // Namespace that all resources for this component are installed into. Overrides for parent feature namespace.
```

Comment:
re: Overrides, this is a single override for the component or multiple overrides?

Reply:
That was not clearly written. I think the new version should be more clear.